docs/source/dashboard.rst

```diff
@@ -61,27 +80,53 @@ indicate the following.
 
     <table>
         <tr>
+            <th colspan="2">For individual worker bars</th>
+            <th colspan="2">For cluster bar</th>
+        </tr>
+        <tr>
+            <td>
+                <div role="img" aria-label="blue square" style="color:rgba(0, 0, 255, 1); font-size: 25px ">&#9632;</div>
+            </td>
+            <td>Memory under target threshold (In above example - 250MiB)</td>
             <td>
                 <div role="img" aria-label="blue square" style="color:rgba(0, 0, 255, 1); font-size: 25px ">&#9632;</div>
             </td>
-            <td>Memory under target (default 60% of memory available) </td>
+            <td>Memory under target threshold (In above example - 250MiB)</td>
         </tr>
         <tr>
             <td>
                 <div role="img" aria-label="orange square" style="color:rgba(255, 165, 0, 1); font-size: 25px ">&#9632;</div>
             </td>
-            <td> Memory is close to the spilling to disk target (default 70% of memory available)</td>
+            <td> Memory is close to the spilling to disk target (default above 50% of memory available)</td>
```

Comment:
There is something going on with then indentation here as it displays a bit off. See the color doesn't start with the sentence, this might get fixed if we use just one column.

Reply:
Okay @ncclementi, I am updating the document and try to add the information in one column only as previously.

Reply:
Updated the description and tried to match as before in one column, and updated the percentage as of now, If find